ui/appui-react/src/appui-react/frontstage/ModalSettingsStage.tsx

```diff
@@ -66,7 +67,7 @@ export class SettingsModalFrontstage implements ModalFrontstageInfo {
   private static noSettingsAvailable = () => new ConditionalBooleanValue(() => 0 === UiFramework.settingsManager.providers.length, [SyncUiEventId.SettingsProvidersChanged]);
 
   public static getBackstageActionItem(groupPriority: number, itemPriority: number) {
-    return BackstageItemUtilities.createActionItem(SettingsModalFrontstage.id, groupPriority, itemPriority, () => FrontstageManager.openModalFrontstage(new SettingsModalFrontstage()),
+    return BackstageItemUtilities.createActionItem(SettingsModalFrontstage.id, groupPriority, itemPriority, () => FrontstageManager.openModalFrontstage(new SettingsModalFrontstage()), // eslint-disable-line deprecation/deprecation
```

Comment:
I have only updated the test apps with the new/moved types. In a package code-base I've decided to use lint ignores everywhere (for consistency), since i.e. we don't want to update public API types.